tests/components/hassio/test_init.py

```diff
@@ -919,10 +928,12 @@ async def test_coordinator_updates(
             },
             blocking=True,
         )
-        assert refresh_updates_mock.call_count == 1
+        assert refresh_updates_mock.call_count == 0
 
-    # There is a 10s cooldown on the debouncer
-    async_fire_time_changed(hass, dt_util.now() + timedelta(seconds=10))
+    # There is a REQUEST_REFRESH_DELAYs cooldown on the debouncer
+    async_fire_time_changed(
+        hass, dt_util.now() + timedelta(seconds=REQUEST_REFRESH_DELAY)
+    )
```

Comment:
Why not use the freezer here?

Reply:
We need the events to fire that are scheduled at that time, the freezer will no do that